roles/advanced-core/advanced_dhcp_server/templates/dhcpd.subnet.conf.j2

```diff
@@ -71,11 +72,12 @@
   }
         {% endif %}
       {% endif %}
-    {% endfor %}
+    {% endfor -%}
 
-    {% if hostvars[host]['bmc'] is defined %}
+    {%- if hostvars[host]['bmc'] is defined %}
```

Comment:
The minus sign should remove whitespace before the block here, it is redundant with `lstrip_blocks: "True"` (1st line of the file)